### PR TITLE
QOL-8056 GitHub actions

### PR DESCRIPTION
**Pull request by ThrawnCA**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -2,4 +2,4 @@ ckantoolkit>=0.0.3
 goodtables==1.5.1
 six>=1.0.0
 pyrsistent<0.17.0
--e git+https://github.com/ckan/ckanext-scheming.git#egg=ckanext-scheming
+-e git+https://github.com/ckan/ckanext-scheming.git@release-2.1.0#egg=ckanext-scheming
```
Comment (duttonw) [inline requirements.txt:5]: please update readme with this being locked (not on master due to xyz)

**Review by duttonw — APPROVED** [review]
Does ahoy local run correctly still?

**Comment by ThrawnCA** [comment]
> Does ahoy local run correctly still?

One of the BDD tests fails locally, but CircleCI works.

**Comment by ThrawnCA** [comment]
> are we still testing 2.4 to 2.7? is this now only 2.8?

I'm inclined to leave those versions to the upstream maintainers.

**Pull request merged**